devtools/maven/src/main/java/io/quarkus/maven/CreateExtensionMojo.java

```diff
@@ -367,6 +367,18 @@
     @Parameter(property = "quarkus.itestParentPath")
     Path itestParentPath;
 
+    /**
+     * Indicates whether to generate a unit test class for the extension
```

Comment:
Apparently, there were `@since` elements added on the others so let's do the same.